week3-todo/script.js

```diff
@@ -0,0 +1,94 @@
+  // Define all UI variable
+  const todoList = document.querySelector('.list-group');
+  const form = document.querySelector('#form');
+  const todoInput = document.querySelector('#todo');
+  const clearBtn = document.querySelector('#clearBtn');
+  const search = document.querySelector('#search');
+  
+  // Load all event listners
+  allEventListners();
+  
+  
+  // Functions of all event listners
+  function allEventListners() {
+      // Add todo event
+      form.addEventListener('submit', addTodo);
+      // Remove and complete todo event
+      todoList.addEventListener('click', removeTodo);
+      // Clear or remove all todos
+      clearBtn.addEventListener('click', clearTodoList);
+      // Search todo event
+      search.addEventListener('keyup', searchTodo);
+  }
+  
+  
+  // Add todo item function
+  function addTodo(e) {
+      if (todoInput.value !== '') {
```

Comment:
We can simply check for the empty strings as `if(!todoInput.value)`

Reply:
okay, I'll keep in mind for further.